cli/cmd/generate.go

```diff
@@ -51,6 +51,28 @@ var (
 		Long:    "Create IaC content for various different cloud environments and configurations",
 	}
 
+	// Azure command
+	generateAzureTfCommand = &cobra.Command{
```

Comment:
this PR is marked to be a feature to the `lwgenerate` Go package, should we move any other change to CLI or any other package to a different PR? or are you going to also add the CLI bits already here?

Reply:
Fair point!
I can drop this out of this PR and move to the future CLI PR :)

Reply:
dropped from this PR 😄